src/Components/SavedSearchAlert/Components/ActiveFilterPills.tsx

```diff
@@ -1,30 +1,19 @@
 import React from "react"
 import { SavedSearchAlertPills } from "./SavedSearchAlertPills"
-import { FilterPill } from "../types"
-import { useActiveFilterPills } from "../useActiveFilterPills"
+import { useActiveFilterPills } from "Components/SavedSearchAlert/useActiveFilterPills"
 import { Flex } from "@artsy/palette"
 
-export interface ActiveFilterPillsProps {
-  defaultPills?: FilterPill[]
-}
-
-const PILL_HORIZONTAL_MARGIN_SIZE = 0.5
+export interface ActiveFilterPillsProps {}
 
-export const ActiveFilterPills: React.FC<ActiveFilterPillsProps> = props => {
-  const { defaultPills = [] } = props
-  const { pills, removePill } = useActiveFilterPills(defaultPills)
+export const ActiveFilterPills: React.FC<ActiveFilterPillsProps> = () => {
+  const { pills, removePill } = useActiveFilterPills()
 
   if (pills.length === 0) {
     return null
   }
 
   return (
-    <Flex
-      flexWrap="wrap"
-      mx={-PILL_HORIZONTAL_MARGIN_SIZE}
-      mb={4}
-      data-testid="artworkGridFilterPills"
-    >
+    <Flex flexWrap="wrap" data-testid="artworkGridFilterPills" gap={1}>
       <SavedSearchAlertPills items={pills} onDeletePress={removePill} />
```

Comment:
This is the only usage of this component currently; probably worth inlining but I'm trying to keep the touch here minimal.